libs/core/src/lib/time/time-column/time-column.component.ts

```diff
@@ -291,6 +303,38 @@ export class TimeColumnComponent<K, T extends SelectableViewItem<K> = Selectable
         }
     }
 
+    /** @hidden
+     * set the time for given item value. used by _numericKeyInputHandler.
+     */
+    _pickTimeOnValue(value: number): void {
```

Comment:
minor: it would feel better if it will be below the method that calls this method, because we read code from top to bottom.